XpringKit.xcodeproj/project.pbxproj

```diff
@@ -9,7 +9,6 @@
 /* Begin PBXBuildFile section */
 		000EF674D02FE63D72B1E1C3 /* AccountBalance.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1585BA85CD7FF784C62F1885 /* AccountBalance.swift */; };
```

Comment:
Not related to this PR at all, but should we keep the xcode project files in git?  I feel like they are super prone to bad merges and have to get regenerated anyway

Reply:
@keefertaylor I have wondered this too.